### PR TITLE
welle.io: update to 2.0

### PR DESCRIPTION
**Pull request by tenzap**
#### Description

Update to 2.0
Compile & install also welle-cli

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.6
Xcode 9.4.1

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by pmetzger** [comment]
This is a relatively complicated change and there's no maintainer, so I'd like us to get another opinion besides mine before merging this.

**Comment by tenzap** [comment]
FYI, I'm the one who created the port

File: multimedia/welle.io/Portfile [hunk 1]
```diff
@@ -5,8 +5,8 @@ PortGroup               qt5 1.0
 PortGroup               github 1.0
 PortGroup               cmake 1.1
 
-github.setup            AlbrechtL welle.io 801e483ce5340cb816cff63743706c245d8c5fe6
-version                 2.0-beta3+git801e483
+github.setup            AlbrechtL welle.io 2.0 v
+github.tarball_from     archive
```
Comment (mf2k) [inline multimedia/welle.io/Portfile]: I believe that you need to add the epoch line so that people will see this update. Can you check on your end?

Comment (tenzap) [reply multimedia/welle.io/Portfile]: Nice catch, epoch added, should be fine now and welle.io is now listed in `port outdated`

**Comment by tenzap** [comment]
BTW, do you know why there is no binary archive here ? http://packages.macports.org/welle.io
Anything wrong with the portfile ?
I thought they would be there

**Comment by mf2k** [comment]
You removed the line `license_noconflict      openssl`, that now prevents us from distributing a binary.

**Comment by tenzap** [comment]
Well, even with line  `license_noconflict openssl` binaries were not distributed.
Honestly I don't know whether this line is legitimate or not. Welle.io doesn't use/link to openssl, only qt5-base does and qt5-base is needed by welle.io.
I think there is no conflict but I'm not sure.

**Comment by mf2k** [comment]
OK. I'm not sure when that is valid anyway. I'll merge as is.

**Pull request merged**